ert_gui/plottery/plots/ensemble.py

```diff
@@ -56,7 +56,12 @@ def _plotLines(self, axes, plot_config, data, ensemble_label, is_date_supported)
         if is_date_supported:
             lines = axes.plot_date(x=data.index.values, y=data, color=style.color, alpha=style.alpha, marker=style.marker, linestyle=style.line_style, linewidth=style.width, markersize=style.size)
         else:
-            lines = axes.plot(data.index.values, data, color=style.color, alpha=style.alpha, marker=style.marker, linestyle=style.line_style, linewidth=style.width, markersize=style.size)
+            index = data.index.values
+            from pandas import DataFrame
+            if isinstance(data, DataFrame):
```

Comment:
Isn't it possible to solve this in the plot API? That API serves as an abstraction over both the old and the new storage.

Reply:
My thinking was that since we are moving towards a brave new world of web visualization, the benefit is not worth the effort?

Reply:
But it sounds like the API is wrong. And the API isn't going away.

Reply:
A new API for transferring data to the _webified_ plots are underway.